horace_core/sqw/@sqw/equal_to_tol.m

```diff
@@ -67,131 +67,148 @@
 % $Revision:: 1759 ($Date:: 2020-02-10 16:06:00 +0000 (Mon, 10 Feb 2020) $)
 
 
-if isa(w1,'sqw') && isa(w2,'sqw')
+if isa(w1, 'sqw') && isa(w2, 'sqw')
     % Check array sizes match
-    if ~isequal(size(w1),size(w2))
-        ok=false; mess='Sizes of sqw object arrays being compared are not equal'; return
+    if ~isequal(size(w1), size(w2))
+        ok = false;
+        mess = 'Sizes of sqw object arrays being compared are not equal';
+        return
     end
     % Check that corresponding objects in the array have the same type
-    for i=1:numel(w1)
-        if is_sqw_type(w1(i))~=is_sqw_type(w2(i))
-            elmtstr=''; if numel(w1)>1, elmtstr=['(element ',num2str(i),')']; end
-            ok=false;
-            mess=['Objects being compared are not both sqw-type or both dnd-type ',elmtstr];
+    for i = 1:numel(w1)
+        if is_sqw_type(w1(i)) ~= is_sqw_type(w2(i))
+            elmtstr = '';
+            if numel(w1) > 1, elmtstr = ['(element ', num2str(i), ')'];
+            end
```

Comment:
as you've split the inline if onto multiple lines; it would be clearer to rewrite this as:
```
if numel(w1) > 1
  elmtstr = ['(element ', num2str(i), ')'];
end
```